### PR TITLE
Implement event-invoice API on admin/sales/status

### LINKED ISSUE #647: Implement invoices api on admin/sales
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

image

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Add event-invoice model 
- Add missing  notification model to user serializer test to prevent build from failing (hence in this PR, separate commit) 
- Implement event-invoice on admin/sales/status

Fixes #647

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-648-open-event-frontend-fossasia.surge.sh

File: app/models/event-invoice.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  /** attributes
+   *
+   */
+
+  status        : attr('string'),
+  identifier    : attr('string'),
+  paypalToken   : attr('string'),
+  transactionId : attr('string'),
+  brand         : attr('string'),
+  createdAt     : attr('moment', { defaultValue: () => moment.tz(detectedTimezone) }),
```
Comment (geekyd) [inline app/models/event-invoice.js]: @CosmicCoder96 Will the frontend create invoices or the server will take care of it?

Comment (CosmicCoder96) [inline app/models/event-invoice.js]: @geekyd The server will take care of this , no post end points for it

Comment (geekyd) [inline app/models/event-invoice.js]: Okay then we do not need to set default values here. Right?

Comment (CosmicCoder96) [inline app/models/event-invoice.js]: makes sense, I will remove it . Nice catch :)

**Comment by CosmicCoder96** [comment]
@geekyd Done, removed the default values

File: app/controllers/admin/sales/status.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    {
+      propertyName     : 'event.name',
+      title            : 'Event',
+      disableSorting   : true,
+      disableFiltering : true
+
+    },
+    {
+      propertyName : 'createdAt',
```
Comment (geekyd) [inline app/controllers/admin/sales/status.js]: @CosmicCoder96 This should be `created-at`

Comment (CosmicCoder96) [inline app/controllers/admin/sales/status.js]: @geekyd Done , thanks !

**Comment by niranjan94** [comment]
Deployment link: https://pr-648-open-event-frontend-fossasia.surge.sh

**Comment by CosmicCoder96** [comment]
@geekyd Done, please review

**Comment by niranjan94** [comment]
Deployment link: https://pr-648-open-event-frontend-fossasia.surge.sh

**Pull request merged**